src/dawn/AST/ASTStringifier.cpp

```diff
@@ -143,6 +143,11 @@ class StringVisitor : public ASTVisitor {
       stmt->getElseStmt()->accept(*this);
     }
   }
+  void visit(const std::shared_ptr<ReductionOverNeighborExpr>& expr) override {
+    if(scopeDepth_ == 0)
+      ss_ << std::string(curIndent_, ' ');
```

Comment:
How does it related to scopeDepth==0?

Reply:
I have no idea, what and how this is relevant for Stmts, but obviously it is not for Expr. THese lines were outdated anyway. Fixed.